src/app/routes/api/v3/billing/__tests__/billing.routes.spec.ts

```diff
@@ -0,0 +1,272 @@
+import { flatten, sortBy, times } from 'lodash'
```

Comment:
Since the endpoint is now `/billings` (note the `s`), the file directory path should reflect that.
Should we change the folder to `/api/v3/billings/` and the route file to `billings.routes.spec.ts`?

Reply:
this makes sense, will change the folder name as proposed for clarity